File: Courses/CSI-Java-2021/MainProject/src/csi/cobian/src/RockPaperScissors.java
/**
 * 
 */
package csi.cobian.src;

import java.util.*;

/**
 * @author carlos.cobian
 *
 */
public class RockPaperScissors {

	/**
	 * @param args
	 */
	public static void main(String[] args) {
		
		String[] choices = {"Rock", "Paper", "Scissors"};
		Random r = new Random();    
		
//		int randomInt = r.nextInt(choices.length)
		String computerChoice = choices[r.nextInt(choices.length)];
		
		Scanner sc = new Scanner(System.in);


		System.out.print("Type your choice: ");
		int inputInt = Arrays.asList(choices).indexOf(sc.nextLine());
		String userChoice = choices[inputInt];

		System.out.println("Computer chose: " + computerChoice);
		System.out.println("User chose: " + userChoice);

		if(computerChoice == userChoice) {
			System.out.println("It's a Tie!");
		}
		else if(computerChoice == choices[0] && userChoice == choices[1]) {
//			Computer chose Rock and user chose Paper
			System.out.println("You Win!");
		}
		//Other else if
		
//		Finally
		else {
			System.out.println("Something went wrong!");
		}
	}

}

File: Repositories/CSI-Python-2021-03/CSI-Andres-Rodriguez/Modules/Module5/RockPaperScissors.py
import random

choices = ["rock", "paper", "scissors"]

MyChoice = input("What is your choice")
print(f"Computer select:{MyChoice})")

computerChoice = random.choice(choices)
print(f"Computer select:{computerChoice}")

if(computerChoice == MyChoice):
    print("tied")
elif(computerChoice == "scissors" and MyChoice == "rock"):
    print("i win")
elif(computerChoice == "rock" and MyChoice == "paper"):
    print("i win")
elif(computerChoice == "paper" and MyChoice == "scissors"):
    print("i win")
elif(computerChoice == "rock" and MyChoice == "scissors"):
    print("i lose")
elif(computerChoice == "paper" and MyChoice == "rock"):
    print("i lose")
elif(computerChoice == "scissors" and MyChoice == "rock"):
    print("i lose")
elif(computerChoice == "scissors" and MyChoice == "scissors"):
    print("tied")
elif(computerChoice == "rock" and MyChoice == "rock"):
    print("tied")
elif(computerChoice == "paper" and MyChoice == "paper"):
    print("tied")
else:
    print("Something went wrong")